src/main.ts

```diff
@@ -19,8 +19,10 @@ async function run(): Promise<void> {
     const owner = ctx.repo.owner
     const repo = ctx.repo.repo
     const pull_number = ctx.payload.pull_request?.number
+    const author = ctx.payload.pull_request?.user?.login
```

Comment:
[nists]

requiredになっているのでこれだけいらなさそうです。
[docs](https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=review_requested#pull_request)
```suggestion
    const author = ctx.payload.pull_request?.user.login
```

Reply:
ご確認いただきありがとうございます!
確認したところ user の値が object or null だったのでオプショナルチェーンで参照したほうが安全かなという感じです!

Reply:
そうなんですね。確認ありがとうございます🙏